quicktest.py

```diff
@@ -53,6 +53,7 @@ def run_tests(self):
                         "context_processors": [
                             "django.contrib.auth.context_processors.auth",
                             "django.contrib.messages.context_processors.messages",
+                            "django.template.context_processors.request",
```

Comment:
Fixing this system check warning:

```
?: (admin.W411) 'django.template.context_processors.request' must be enabled in DjangoTemplates (TEMPLATES) in order to use the admin navigation sidebar.
```

This has been present since Django 3.2, from this commit: https://github.com/django/django/commit/d522b51c401429c169d88742178a9b3777903d9e